solarforecastarbiter/metrics/tests/test_preprocessing.py

```diff
@@ -534,6 +645,16 @@ def test_process_forecast_observations_resample_fail(
         report.report_parameters.object_pairs, filters, data, 'MST')
     assert len(processed_fxobs_list) == 0
     assert logger.error.called
+    for proc_fxobs in processed_fxobs_list:
+        assert isinstance(proc_fxobs, datamodel.ProcessedForecastObservation)
+        assert all(isinstance(vr, datamodel.ValidationResult)
+                   for vr in proc_fxobs.validation_results)
+        assert all(isinstance(pr, datamodel.PreprocessingResult)
+                   for pr in proc_fxobs.preprocessing_results)
+        assert isinstance(proc_fxobs.forecast_values, pd.Series)
+        assert isinstance(proc_fxobs.observation_values, pd.Series)
+        pd.testing.assert_index_equal(proc_fxobs.forecast_values.index,
+                                      proc_fxobs.observation_values.index)
```

Comment:
codecov says these lines are not hit, but I don't understand why.

https://codecov.io/gh/SolarArbiter/solarforecastarbiter-core/compare/d123e88062d0debdb01516193b246176c3c95cb0...61c4b444cbc92ccc1ba102fd62e7e95da553b10e/diff#D2-648...657

Reply:
Removed the for loop